add-geolocation/test.json

```diff
@@ -0,0 +1,3 @@
+{
+  "id": 1
```

Comment:
here you have to to have the IP as a part of the event, the geolocation enrichment is not made for the producer IP but for an IP from within the event